src/test/groovy/org/ods/doc/gen/leva/doc/fixture/ProjectRepositoryFixture.groovy

```diff
@@ -0,0 +1,49 @@
+package org.ods.doc.gen.leva.doc.fixture
+
+class ProjectRepositoryFixture {
```

Comment:
This file does not scale to more projects

Reply:
Yours neither. This makes it possible to test ordgp project.
If we remove them, tests fail.
It is obvious that we'll have to fix your work in the future, but right now we can live with it.

Reply:
No, this is not true. This data does not scale at all